game/blocks.py

```diff
@@ -68,7 +68,6 @@ def __init__(self, name, tex_coords):
         self.name = name
         self.tex_coords = tex_coords
 
-
```

Comment:
This whitespace should remain for PEP8 compliance.

Reply:
What is the violation?

Reply:
There should be two blank lines after a class or function definition.

Reply:
Oh! I was actually not aware of that one! Thanks!